frontend/common/Debug.tsx

```diff
@@ -5,8 +5,8 @@ import { PageHeader } from '../../framework';
 import { useBulkActionDialog } from '../../framework/BulkActionDialog';
```

Comment:
What does this file do?  The name of it 'Debug' is not very descriptive.  Do we need it?

Reply:
It is the Debug tab in the UI which only shows up in a development build